Common/cpp/SharedItems/ShareableValue.cpp

```diff
@@ -206,7 +206,19 @@ jsi::Value ShareableValue::toJSValue(jsi::Runtime &rt) {
       } else {
         // function is accessed from a different runtme, we wrap function in host func that'd enqueue
         // call on an appropriate thread
+        
         auto module = this->module;
+        auto warnFunction = [module](
+            jsi::Runtime &rt,
+            const jsi::Value &thisValue,
+            const jsi::Value *args,
+            size_t count
+            ) -> jsi::Value {
+          module->errorHandler->setError("tried to synchronously call function from a diffrent thread");
```

Comment:
We should suggest a solution here either by stating it in the error message or providing a link where they can see how to resolve the problem.

Solution is:
a) if you want to synchronously execute this method, mark it as a worklet
b) if you want to execute this method on the JS thread, wrap it using `runOnJS`